src/pages/private/index.js

```diff
@@ -30,6 +35,7 @@ function Private () {
         path={PRIVATE_HOMEPAGE}
         component={() => <Homepage user={user}/>}
```

Comment:
Should be used another ways to provide logged user to into Route nested components ?